web_ui/src/pages/project-details/components/project-models/train-model-dialog/advanced-settings/data-management/tiling/tiling.component.tsx

```diff
@@ -15,29 +18,105 @@ import styles from './tiling.module.scss';
 
 interface TilingProps {
     tilingParameters: ConfigurationParameter[];
+    onUpdateTrainingConfiguration: (
+        updateFunction: (config: TrainingConfiguration | undefined) => TrainingConfiguration | undefined
+    ) => void;
 }
 
-const getTilingMode = (tilingParameters: ConfigurationParameter[]): TILING_MODES => {
-    const adaptive = tilingParameters.find((parameter) => parameter.key === 'enable_adaptive_params');
-    const enablingTiling = tilingParameters.find((parameter) => parameter.key === 'enable_tiling');
+const ADAPTIVE_TILING_PARAMETER = 'adaptive_tiling';
+const ENABLE_TILING_PARAMETER = 'enable';
+
+const getAdaptiveTilingParameter = (tilingParameters: ConfigurationParameter[]) => {
```

Comment:
Possible suggestion since we're doing the same `.find()` and then `!isBoolParameter` here, we could use a helper,
```
const findParameter = (key: string) => (parameter: ConfigurationParameter) => parameter.key === key && !isBoolParameter(parameter)
// tilingParameters.find(findParameter(ADAPTIVE_TILING_PARAMETER))
```

or something similar.

I'm not sure though if it's worth the extra abstraction so I'm also fine with keeping it as is. The main reason I was thinking of it is that I didn't understand why we need to do `!isBoolParameter`?